packages/node/src/types.ts

```diff
@@ -260,8 +261,8 @@ export type EVMRequestCreatedLog = EVMTemplateRequestCreatedLog | EVMFullApiRequ
 
 export interface EVMRequestFulfilledLog extends EVMEventLogMetadata {
   readonly parsedLog:
-  | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFulfilled'>>
-  | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFailed'>>;
+    | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFulfilled'>>
+    | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFailed'>>;
```

Comment:
I'm sorry about this, I have no idea why it's still moving. I'd have to figure it out so it won't happen again :man_facepalming: